Client/Extensions/AppearanceExtensions.swift

```diff
@@ -67,9 +67,6 @@ extension Theme {
         InsetButton.appearance(whenContainedInInstancesOf: [SearchSuggestionPromptView.self]).appearanceTextColor = colors.tints.home
         
         // Overrides all views inside of itself
-        // According to docs, UIWindow override should be enough, but some labels on iOS 13 are still messed up without UIView override as well
```

Comment:
The comment is so contradictory I wonder why did they add it to window before :)

Reply:
Yeah I am not sure, keeping it unfortunately completely breaks SwiftUI re-render after color scheme changes